dusty/commands/repos.py

```diff
@@ -21,10 +21,19 @@ def list_repos():
                        repo.override_path if repo.is_overridden else ''])
     log_to_client(table.get_string(sortby='Short Name'))
 
+def nfs_path_exists(path):
+    split_path = path.lstrip('/').split('/')
+    recreated_path = '/'
+    for path_element in split_path:
+        if path_element not in os.listdir(recreated_path):
```

Comment:
This does in fact return case-sensitive results, right? Any way to do a unit test with it against actual files on an HFS filesystem?